src/is_scene.py

```
@@ -211,22 +207,19 @@ async def is_scene(self, video: str, meta: Meta, imdb: int | None = None, lower:
 
                             return release_name, True, imdb
                         else:
-                            if meta.debug:
-                                logger.debug("[yellow]SRRDB: No match found with lower/tag search")
+                            logger.debug("[yellow]SRRDB: No match found with lower/tag search")
                             return video, scene, imdb
 
                     except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)